tutorials/animation/introduction.rst

```diff
@@ -365,7 +365,7 @@ for each track.
 
 If AnimationPlayer's **Reset On Save** property is set to ``true``,
 the scene will be saved with the effects of the reset animation applied
-(as if it had been seeked to time ``0.0``).
+(as if it had been sought to time ``0.0``).
```

Comment:
The term in this case is `seeked` and is established at least in Godot terminology

Reply:
removed the change